README.md

```diff
@@ -1 +1,2 @@
 # hw01
+# An amazing project
```

Comment:
Good job, Sandy. This was short, sweet, and to the point. I think you could have spiced up your work a little bit by changing # An amazing project to # _An amazing project_

Reply:
And I could've improved my comment by using ### _An amazing project_